tests/integration/test_hnsw_psql.py

```diff
@@ -98,3 +99,51 @@ def test_basic_integration(docker_compose, get_documents):
             status
         )
         assert last_sync > last_sync_timestamp
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_replicas_integration(docker_compose, get_documents):
+    emb_size = 10
+    LIMIT = 10
+    NR_SHARDS = 2
+    docs = DocumentArray(get_documents(nr=100, emb_size=emb_size))
+
+    uses_with = {
+        'dim': emb_size,
+        'startup_sync': False,
```

Comment:
What would actually happen, if this is set to `True`? Would it explode or just nothing? I would expect just nothing...

Reply:
nothing. removed